docs/source/deployment/amazon_emr_serverless.md

```diff
@@ -194,7 +194,7 @@ Enter the respective values in the placeholders above. For example, use the ARN
 
 ## FAQ
 
-### How is the approach defined here different from the approach in ["Seven steps to deploy Kedro pipelines on Amazon EMR"](https://kedro.org/blog/how-to-deploy-kedro-pipelines-on-amazon-emr) on the Kedro blog?
+### How is the approach defined here different from the approach in ["Seven steps to deploy Kedro pipelines on Amazon EMR"](https://api/kedro.org/blog/how-to-deploy-kedro-pipelines-on-amazon-emr) on the Kedro blog?
```

Comment:
Another search & replace gone wrong

Reply:
Curses, it's not failing on local build, of course, because I'm not link checking. Will fix these (and my script)!